Sources/SpeziViews/Views/Text/MarkdownView.swift

```diff
@@ -24,6 +24,23 @@ import SwiftUI
 /// )
 /// ```
 public struct MarkdownView: View {
+    public enum Error: LocalizedError {
+        case markdownLoadingError
+        
+        public var errorDescription: String? {
+            "Failed to load the markdown text."
+        }
+        
+        public var recoverySuggestion: String? {
+            "Please check the content of the markdown text."
+        }
+
+        public var failureReason: String? {
+            "The system wasn't able to parse the given markdown text, indicating an invalid markdown text."
+        }
```

Comment:
It would be great if these values can be localized as they might be surfaced to users 👍